src/models/Projection.ts

```diff
@@ -1,5 +1,12 @@
 import { RegionSummaryWithTimeseries, Timeseries } from 'api';
 
+/**
+ * We truncate (or in the case of charts, switch to a dashed line) the last
+ * seven days of r(t) data because it is susceptible to continued change as we
+ * get future data points.
+ */
+export const RT_TRUNCATION_DAYS = 7;
```

Comment:
@pnavarrc I exported this constant so perhaps you can pull it in for your charts work?